Simplenote/Classes/SPInteractiveTextStorage.h

```diff
@@ -6,6 +6,6 @@ extern NSString *const SPHeadlineTokenName;
 
 @interface SPInteractiveTextStorage : NSTextStorage
 
-@property (nonatomic, copy) NSDictionary *tokens; // a dictionary, keyed by text snippets, with attributes we want to add
+@property (nonatomic, copy) NSDictionary<NSString *, NSDictionary<NSAttributedStringKey, id> *>* tokens;
```

Comment:
Incredibly nitpicky of me, but with code style, shouldn't the asterisk cuddle tokens? 
```
@property (nonatomic, copy) NSDictionary<NSString *, NSDictionary<NSAttributedStringKey, id> *> *tokens;
```

Reply:
You're right!! updating, thank you!